### PR TITLE
Add recipe for bioconductor-bayespeak 1.24.0.

### PR DESCRIPTION
**Pull request by acaprez**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

ping @bioconda/core 
I've been away from the project for a bit and see there's an issue with R 3.3 and updating Bioconductor packages.  Is it ok to add new Bioconductor packages in the meantime?

### CODE REVIEW

**Comment by bgruening** [comment]
It's ok! Are you available to help us with the R3.3 migration, basically rebuild every R package :)

**Comment by bgruening** [comment]
Looks good to me. Feel free to merge!

File: recipes/bioconductor-bayespeak/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  md5: 821ed8cbee83a5b8dfd7530306d9e3ee
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  build:
+    - bioconductor-iranges
+    - 'r >=2.14'
```
Comment (bgruening) [inline recipes/bioconductor-bayespeak/meta.yaml]: Does this needs to be `r` or `r-base`?

Comment (acaprez) [inline recipes/bioconductor-bayespeak/meta.yaml]: That was just what the `bioconductor_skeleton.py` script generated.  I can change it to `r-base` if that's the preferred convention.

**Comment by acaprez** [comment]
> It's ok! Are you available to help us with the R3.3 migration, basically rebuild every R package :)

I might be, depending on when it's planned for.  The next month or so is quite busy for me with other things unfortunately.

**Pull request merged**